Mlem/App/Views/Root/Tabs/Inbox/InboxView.swift

```diff
@@ -108,8 +95,31 @@ struct InboxView: View {
     var content: some View {
         FancyScrollView {
             LazyVStack(spacing: 0, pinnedViews: [.sectionHeaders]) {
+                FeedHeaderView(
+                    feedDescription: .init(
+                        label: "Inbox",
+                        subtitle: "Replies, mentions and messages",
+                        color: \.inbox,
```

Comment:
This line causes build failures for me on Xcode 15,

```suggestion
                        color: { $0.inbox },
```